src/components/Navbar.js

```diff
@@ -39,6 +37,33 @@ function Navbar() {
 							"md:flex md:flex-row md:justify-center md:w-auto md:relative xs:hidden xs:flex-none"
 						}
 					>
+						<div className="p-2 mx-2 dropdown inline-block relative">
+							<button>
+								Courses
+								<FontAwesomeIcon
+									className="ml-2"
+									icon={faAngleDown}
+								/>
+							</button>
+							<ul className="dropdown-menu absolute hidden pt-2 bg-white shadow-2xl">
+								<Link to="/courses/python/beginner">
+									<li className="rounded-t py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
```

Comment:
The `rounded-t` and `rounded-b` classes can't be applied to `ul`? Seems like extra work to apply them to first and last child. Try putting `rounded-lg` on `ul`